record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestParameterizedBuilder.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.soabase.recordbuilder.test;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
+import java.util.OptionalLong;
+
+class TestParameterizedBuilder {
+    @Test
+    void testNoSubclass() {
+        ParameterizedBuilderBuilder<?> builder = ParameterizedBuilderBuilder.builder();
```

Comment:
It feels odd to have a static builder in a parameterized builder. It should likely not be added for parameterized and force use of the constructors.

Reply:
Is the no-subclass even useful? We could make the builder `abstract` when parameterized.